samples/csharp/getting-started/Ranking_PersonalizedSort/PersonalizedRanking/PersonalizedRanking.csproj

```diff
@@ -0,0 +1,21 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Remove="DataStructures\HotelData.cs" />
```

Comment:
These can all be removed, can't they? I don't see a file `Program_old.cs` or `Mapper.cs`

Reply:
yep - i missed that - thanks! #resolved

Reply:
#resolved